applications/serial_lte_modem/doc/slm_description.rst

```diff
@@ -1645,6 +1662,7 @@ Complete the following steps to test the functionality provided by the :ref:`SLM
 Testing FTP AT commands
 -----------------------
 
+Note that these commands are available only if :option:`CONFIG_SLM_FTPC` is defined.
```

Comment:
A note like this must also be included in the (new) GPS section.

Reply:
Added

Reply:
I can't see this change, did you maybe not push it?

Reply:
It's a shame that I committed without including the change.
Now fixed